### ISSUE TITLE
UAC Causes Remote Desktop to freeze

### ISSUE THREAD

**Issue by xuxuedong** [MaxXor/xRAT#733]
stuck when i open task manager on remote desktop
at that moment, the cpu consume is larger than the normal
![qq 20181209092428](https://user-images.githubusercontent.com/9376976/49692371-479cf380-fb94-11e8-9f74-f59d7afd8344.png)

tested server os: win10_pro_x64_cn
test client os: win8.1_pro_x64_cn

**Comment by trollmad3** [comment]
Could you post your log so we can further help?

**Comment by xuxuedong** [comment]
log? windows event log? there is none log file within the current directory

**Comment by xuxuedong** [comment]
as you see, from "下载46% "to "下载48%", it isn't stuck, just unresponsible when i click after start task manager
![sbvxy 9nz l49 vwtmb0l](https://user-images.githubusercontent.com/9376976/49786016-4bc83d00-fd5d-11e8-879f-16c45cb289e1.png)
![qq 20181211155543](https://user-images.githubusercontent.com/9376976/49786027-5551a500-fd5d-11e8-8948-f9f00feae71b.png)

**Comment by xuxuedong** [comment]
as i see, it isn't stuck, just unresponsible when i click after started task manager

**Comment by xuxuedong** [comment]
as i see, it isn't stuck, the picture is changing, just unresponsible when i click after started task manager

**Comment by xuxuedong** [comment]
but when i tested:
server os: win10_pro_x64_cn
client os: xp_pro_x86_cn
i use remote desktop to operate xp, it's responsible when i click after i started task manager

in addition, i found the cpu consume is normal

**Comment by trollmad3** [comment]
So is this issue fixed?

**Comment by MaxXor** [comment]
@trollmad3 No, it's not on Windows 8.1.

**Comment by trollmad3** [comment]
When I get home, I will create a windows 8.1 virtual machine and see if I could do anything to fix this.

**Comment by mystyq** [comment]
This behavior also occurs when opening chrome browser (temporarily).

**Comment by mystyq** [comment]
I believe the problem is related to the `mouse_event` WinAPI calls.  If you run the task manager as administrator it seems to work.  `mouse_event` is deprecated and I wonder if switching to `SendInput` calls would fix this.

**Comment by xuxuedong** [comment]
@mystyq yeah, you are right, i tested the client run as admin on win8.1, it's no problem
